spec/spec_helper_acceptance_local.rb

```diff
@@ -34,9 +34,12 @@ def print_parsing_errors
   end
   c.before :suite do
     # Make sure selinux is disabled so the tests work.
-    LitmusHelper.instance.run_shell('setenforce 0', expect_failures: true) if %r{redhat|oracle}.match?(os[:family])
+    LitmusHelper.instance.run_shell('setenforce 0', expect_failures: true) if %r{redhat}.match?(os[:family])
```

Comment:
Would this be more consistent?
```suggestion
    LitmusHelper.instance.run_shell('setenforce 0', expect_failures: true) if os[:family] == 'RedHat'
```